pkg/agent/flowexporter/connections/conntrack_connections.go

```diff
@@ -0,0 +1,251 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package connections
+
+import (
+	"encoding/binary"
+	"fmt"
+	"time"
+
+	"github.com/vmware/go-ipfix/pkg/registry"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter"
+	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter/flowrecords"
+	"github.com/vmware-tanzu/antrea/pkg/agent/metrics"
+	"github.com/vmware-tanzu/antrea/pkg/agent/openflow"
+	"github.com/vmware-tanzu/antrea/pkg/querier"
+)
+
+var serviceProtocolMap = map[uint8]corev1.Protocol{
+	6:   corev1.ProtocolTCP,
+	17:  corev1.ProtocolUDP,
+	132: corev1.ProtocolSCTP,
+}
+
+type ConntrackConnectionStore interface {
+	// Run enables the periodical polling of conntrack connections at a given flowPollInterval.
+	Run(stopCh <-chan struct{})
+	// Poll calls into conntrackDumper interface to dump conntrack flows. It returns the number of connections for each
+	// address family, as a slice. In dual-stack clusters, the slice will contain 2 values (number of IPv4 connections first,
+	// then number of IPv6 connections).
+	Poll() ([]int, error)
```

Comment:
These two methods are not required as they are not used in any unit tests right? I should have removed this in the past, but left them there thinking we may use the interface for more unit tests. 
I feel better removing the unnecessary methods from the interface that are not used in tests.

Reply:
Makes sense to me. Removed them